### ISSUE TITLE
Using ngModel in MDTextfield

### ISSUE THREAD

**Issue by williamreetz** [Akylas/nativescript-material-components#8]
Is there a way to use "ngModel"? I tried it, but it doesn't work.
`<MDTextField [(ngModel)]='model.mytext'></MDTextField>`

**Comment by farfromrefug** [comment]
It was working before. Will bring that back.

**Comment by farfromrefug** [comment]
Actually can you try importing ```MaterialTextfieldModule``` from ```nativescript-material-textfield/angular/material-textfield.module```? This should work.

Be careful we will change the path in the future releases

**Comment by williamreetz** [comment]
I can't import MaterialTextfieldModule. I will use textChange event for the moment.

**Comment by farfromrefug** [comment]
@williamreetz that's the issue. need to make a new release

**Comment by t0ms3n** [comment]
Hi there,  
I'm using v2.2.17 and can not get the MDTextfield to work with [(ngModel)]. Furthermore I would love to use it with reactive forms.

It's reproducable with the default HelloWorld template. I have imported the 
import { MaterialTextfieldModule } from 'nativescript-material-textfield/angular'; at the HomeModule and defined the following imports:
imports: [
        NativeScriptCommonModule,
        NativeScriptFormsModule,
        MaterialTextfieldModule,
        HomeRoutingModule
    ],

In the home.component.html I have defined:
<MDTextField
                 [(ngModel)]="title"
                 hint="i am an hint"
                 helper="help me!"
                 returnKeyType="next"></MDTextField>

As soon as the app starts the following error is thrown:
ERROR Error: No value accessor for form control with unspecified name attribute

Is there any known issue with this or what can I do to make it working again?
